src/pages/axa-ch-main/commercial-hero-banner.js

```diff
@@ -0,0 +1,20 @@
+import { html } from 'lit-html';
+
+export default html`
+  <axa-commercial-hero-banner
+    variant="dark"
+    imagesource="https://brandcenter.axa.ch/m/4df4627fddd0534a/WIDE_1920_560-AXA_DE_DANNY_web.jpg"
+  >
+    <h2 slot="category">
+      Jetzt Haushalt und Privathaftpflicht versichern und profitieren
+    </h2>
+    <h1 slot="title">Bis zu CHF 200.– Rabatt</h1>
+    <axa-button-link
+      href="https://axa.ch"
```

Comment:
Is it possible to not give a link? Our showcase should not be left by clicking such a button.

Even the links in header and footer should'nt be removed in my opinion.

Reply:
Sure, I used # on other links, will do that here too, thanks!